dolphinscheduler-storage-plugin/dolphinscheduler-storage-api/src/main/java/org/apache/dolphinscheduler/plugin/storage/api/StorageType.java

```diff
@@ -25,7 +25,9 @@ public enum StorageType {
     HDFS(1, "HDFS"),
     OSS(2, "OSS"),
     S3(3, "S3"),
-    GCS(4, "GCS");
+    GCS(4, "GCS"),
+
```

Comment:
if we add it, we can add some document like 

```text
Note that there is a duplicate empty file <no name> since Azure blob storage does not support having empty folders. But this does not affect the file display in the ds resource center
```

in our document

Reply:
> if we add it, we can add some document like
> 
> ```
> Note that there is a duplicate empty file <no name> since Azure blob storage does not support having empty folders. But this does not affect the file display in the ds resource center
> ```
> 
> in our document

Thanks for your suggestion, I'll add it